webextensions/javascript-apis.json

```diff
@@ -3453,7 +3453,7 @@
                     "version_added": "48.0"
                   },
                   "opera": {
-                    "version_added": false
+                    "version_added": true
```

Comment:
The Opera docs say, under the `tabs` docs,that "Opera currently does not support the InjectDetails...methods", although they also say that `extensions` is fully supported.

Since I don't see how you can support `tabs.executeScript` without `InjectDetails`, then I suppose it must be supported, so this change is probably right.